qemu/tests/cfg/slof_user_interface.cfg

```diff
@@ -11,7 +11,7 @@
     boot_order = cdn
     boot_strict = off
     usb_devices += ' kbd1'
-    usb_type_kbd1 = 'usb-kbd'
+    usbdev_type_kbd1 = 'usb-kbd'
```

Comment:
@YongxueHong  Could you help review the 4 slof related cases ? Thanks!

Reply:
> @YongxueHong Could you help review the 4 slof related cases ? Thanks!

Hi @MiriamDeng 
Would you like to help review those? Thanks.

Reply:
I'm afraid that there's something wrong, ran the related jobs on ppc and i got
(status: 1,    output: "qemu-kvm: -device qemu-xhci,id=usb-kbd1,bus=usb1.0,port=2: Device 'qemu-xhci' can't go on usb-bus bus")